nginx_stage/lib/nginx_stage/generators/nginx_clean_generator.rb

```diff
@@ -59,8 +62,9 @@ class NginxCleanGenerator < Generator
           next if (user && user != u.to_s)
           pid_path = PidFile.new NginxStage.pun_pid_path(user: u)
           socket = SocketFile.new NginxStage.pun_socket_path(user: u)
```

Comment:
It seems that this would still run `lsof` as the SocketFile initializer runs `@processes = get_processes`, although the output itself is not used. There is the same case for nginx_show_generator.

Reply:
Thanks, this is done. I believe @CSC-swesters also wanted it removed, which it is now.